src/asset-catalog/asset-search-view.js

```diff
@@ -218,44 +221,58 @@ class AssetSearchView extends mixinBehaviors([AppLocalizeBehavior], PolymerEleme
                 vaadin-grid {
                     flex-grow: 1;
                 }
+                
+                paper-checkbox {
+                    align-self: center;
+                    border: 1px solid var(--egeria-primary-color);
+                    padding: 8px 16px;
+
+                    --paper-checkbox-checked-color: var(--egeria-primary-color);
+                    --paper-checkbox-checked-ink-color: var(--egeria-primary-color);
+                    --paper-checkbox-unchecked-color: var(--egeria-secondary-color);
+                    --paper-checkbox-unchecked-ink-color: var(--egeria-secondary-color);
+                    --paper-checkbox-label-color: var(--egeria-primary-color);
+                    --paper-checkbox-label-spacing: 0;
+                    --paper-checkbox-margin: 3px 8px 3px 0;
+                }
 
             </style>
-
+            
             <token-ajax id="tokenAjax" last-response="{{items}}"></token-ajax>
             <token-ajax id="tokenAjaxTypes" last-response="{{supportedTypes}}"></token-ajax>
 
 
             <iron-form id="searchForm">
                 <form method="get">
                     <iron-a11y-keys keys="enter" on-keys-pressed="_search"></iron-a11y-keys>
-                    <div>
-                        <div style="display: inline-block; vertical-align: top">
-                            <div style="width: 200pt; display: inline-block;">
-                                <paper-input class="custom" id="searchField"
-                                             label="Search" value="{{q}}"
-                                             no-label-float
-                                             required
-                                             minlength="2"
-
-                                             autofocus>
-                                    <iron-icon icon="search" slot="prefix" class="icon"
-                                               style="background: transparent"></iron-icon>
-                                </paper-input>
-                            </div>
-
-                            <vaadin-button id="searchSubmit" theme="primary" on-tap="_search">
-                                <iron-icon id="search" icon="search"></iron-icon>
-                            </vaadin-button>
-                        </div>
-
+                    <div style="text-align: center;margin: 20px;"> 
```

Comment:
use bem class here such as ".align-center, .m20" and compose them in the class attribute